network-api/networkapi/buyersguide/models.py

```diff
@@ -144,6 +233,42 @@ class Product(models.Model):
         blank="True",
     )
 
+    update1 = models.ForeignKey(
```

Comment:
This also feels like a weird way to implement this. @gideonthomas do you happen to know of a DRYer, simple way to do this?